jenkins-build.sh

```diff
@@ -426,7 +470,7 @@ fi
 while [[ $retry -le $max_retries ]]
 do
   statuss3=0
-  $USE_SUDO ./dev-starts3.sh $fake_params $callgraph_cmd $valgrind_memcheck_cmd
+  $USE_SUDO ./dev-starts3.sh $num_instances $fake_params $callgraph_cmd $valgrind_memcheck_cmd
```

Comment:
dev-start.sh and dev-stop.sh have a const "MAX_S3_INSTANCES_NUM=20" and $num_instances should be less than MAX_S3_INSTANCES_NUM.
I think at least some info about this restriction should be printed

Reply:
Discussed with @dsurnin  -- he will take it up in separate PR.